tutorials/nel_emerson/project.yml

```diff
@@ -30,6 +30,8 @@ assets:
   - dest: "assets/emerson_input_text.txt"
     description: "The original input text"
     checksum: "b5bc61375680396017337e3ebe830ada"
+  - dest: "configs/${vars.config}"
+    description: "The config file for training"
```

Comment:
Do we want to put configs in assets? I feel like this could potentially be misleading and it doesn't really do anything (except for making it show up in the readme).

Reply:
Not sure what the convention should be. Either way's fine by me.